atlp-rwanda/octopus-bn-backend
# #172527906 Tweak the profile endpoint

### What does this PR do?

- Enable ```one way manager automatic assignment``` for users while updating their profile
- Automate the assignment using the randomiser

### How should this be manually tested?

- Clone the repo
- Run ``` npm install``` to install all dependencies
- Run ``` npm run test``` to check if all unit tests run safely
- install ```Postman```  and run ``` npm run dev``` to test this endpoint: ```localhost:5000/api/v1/auth/profile-settings```. ``PS``: Make sure that you create an account and pass the token in the header for the server to be aware of who.

### Implementation plan
N/A
### Stories associated with PR

[#172527906](https://www.pivotaltracker.com/story/show/172527906)

### Screenshots if any
<img width="1156" alt="Screen Shot 2020-04-28 at 01 02 50" src="https://user-images.githubusercontent.com/51243858/80429280-01147a00-88ec-11ea-8289-6cab596c4366.png">
